modules/checkout/commerce_checkout.info.yml

```diff
@@ -10,3 +10,8 @@ dependencies:
 config_devel:
   install:
     - commerce_checkout.commerce_checkout_flow.default
+    - core.entity_view_display.commerce_order.default.summary
```

Comment:
Removing, then adding it in a followup to both cart & order summary once the new area plugin is done.